paint.js

```diff
@@ -740,7 +740,7 @@ PaintCanvasMorph.prototype.floodfill = function (sourcepoint) {
         ctx = this.paper.getContext("2d"),
         img = ctx.getImageData(0, 0, width, height),
         data = img.data,
-        stack = [Math.round(sourcepoint.y) * width + sourcepoint.x],
+        stack = [Math.round(Math.round(sourcepoint.y) * width + sourcepoint.x)],
```

Comment:
Do we need to round twice? I'm not sure why it's important to round `.y`?

Also, is it ever possible for this to include a pixel that's out of bounds if you were to select the lower right corner and things rounded up? Perhaps that's a super weird edge case, but I might write:
`parseInt(sourcepoint.y * width + sourcepoint.x)`. I don't know if that would make the normal case more likely to be 1px off of where someone clicked?